Formula/borgbackup.rb

```diff
@@ -0,0 +1,32 @@
+class Borgbackup < Formula
+  include Language::Python::Virtualenv
+
+  desc "Deduplicating archiver with compression and authenticated encryption"
+  homepage "https://borgbackup.readthedocs.io/en/stable/"
```

Comment:
just use https://borgbackup.org/ ?

the docs are linked from there, but there is also other information.

Reply:
OK. Used that site. I used the other URL before because the current Cask uses it.